docs/index.md

```diff
@@ -1,22 +1,32 @@
 ---
-title: Welcome
+title: " "
 description: 3D City Database v5 documentation welcome page
 ---
 
-#### :simple-rocket: Welcome to the brand new 3D City Database `v5` documentation! :simple-rocket:
+## :simple-rocket: Welcome to the 3D City Database `v5` documentation :simple-rocket:
+
+[![3DCityDB logo](./assets/img/logos/3dcitydb-logo-long.png){ width="400" }](https://www.3dcitydb.org){target="blank"}
+/// caption
+///
 
 The 3D City Database (3DCityDB) is a free and open source package consisting of a database schema and a set of software
-tools to import, export, manage, analyse, and visualize virtual 3D city models on top of a spatial database system. This
-user manual provides the documentation of the 3DCityDB `v5` and its components.
+tools to import, export, manage, analyse, and visualize virtual 3D city models on top of a spatial database system. This is version `1.0.0` of the
+user manual. It provides the documentation of the 3DCityDB and its components in the following versions.
 
-The content of this documentation is all new! If you encounter a mistake, miss some content, or for any other input,
-please get in touch on [GitHub discussions](https://github.com/orgs/3dcitydb/discussions). We are happy for any feedback
-we can use to improve the documentation.
+| Component                           | Version                                |
+|-------------------------------------|----------------------------------------|
+| 3D City Database                    | `5.1.0`                                |
+| citydb-tool                         | `1.1.0`                                |
```

Comment:
A bit tricky. These versions will be included in version 1.1 of the documentation. Version 1.0.0 is about 3DCityDB 5.0.0 and citydb-tool 1.0.0. I guess this needs to be adapted manually after merging.

Reply:
Yes, I'll create a `1.0` release adding the table manually with the correct versions.